_docs/quickstarts/scanner/default-rules.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Default Rules
+order: 106
+---
+
+# {{ page.title }}
+
+Forseti Scanner is preconfigured with some default rules. When any of these rules' conditions are met, 
```

Comment:
Forseti Scanner has default rules that create a [violation]({% link _docs/quickstarts/scanner/policy-violations.md %}) when their conditions are met.